### PR TITLE
Force old recipes when classic is loaded

### PR DESCRIPTION
**Pull request by Chocohead**
Should fix https://github.com/xbony2/Nuclear-Control/issues/110.

### CODE REVIEW

**Comment by xbony2** [comment]
Although it should be old per default, it should still be possible to change the mode.

**Comment by Chocohead** [comment]
How about now?

File: src/main/java/shedar/mods/ic2/nuclearcontrol/IC2NuclearControl.java [hunk 1]
```diff
@@ -173,37 +173,42 @@ public void init(FMLInitializationEvent event) {
...
 		crossClassic = new CrossIndustrialCraft2Classic();
 
-		if (recipes.equalsIgnoreCase("normal"))
-			RecipesNew.addRecipes();
-
-		if (recipes.equalsIgnoreCase("old")) {
-			logger.error("Old recipes deprecated! Many recipes may be missing.");
+		if (crossClassic.doesIC2ClassicExist()) {
+			if (recipes.equalsIgnoreCase("normal-force")) {
+				logger.info("Loading normal recipes with IC2 Classic may prevent certain recipes working")
```
Comment (xbony2) [inline src/main/java/shedar/mods/ic2/nuclearcontrol/IC2NuclearControl.java]: Broah, pretty sure you need semicolons in java :P

Comment (Chocohead) [inline src/main/java/shedar/mods/ic2/nuclearcontrol/IC2NuclearControl.java]: Any good editor would've automatically added them in *stares at Github*

**Comment by xbony2** [comment]
#RubyMasterrace

**Pull request merged**